bin/heroku-php-apache2

```diff
@@ -387,21 +389,43 @@ echo -e "\n[global]\nlog_level = notice" >> "$fpm_config_tmp"
 fpm_command=( php-fpm --pid "$fpm_pidfile" --nodaemonize -y "$fpm_config_tmp" ${php_config:+-c "$php_config"} )
 httpd_command=( httpd -D NO_DETACH -c "Include $httpd_config" )
 
-mlib="/sys/fs/cgroup/memory/memory.limit_in_bytes"
-if [[ -f "$mlib" ]]; then
-	[[ $verbose ]] && echo "Reading available RAM from '$mlib'" >&2
-	ram="$(cat "$mlib")"
-else
-	[[ $verbose ]] && echo "No '$mlib' with RAM info found" >&2
-	ram="512M"
-	echo "Assuming RAM to be ${ram} Bytes" >&2
+ram=
+# a hard maximum we never want to exceed, e.g. in case of silly values
+memory_maximum=$(( 8*1024*1024*1024*1024 ))
```

Comment:
I would like a unit here. Would be good to either have it in the variable name or a comment or extracted as a variable:

```suggestion
terabyte=$((1024*1024*1024*1024))
memory_maximum=$((8*terabyte))
```

This is assuming the input `-m` takes bytes, and is supposed to be 8 TB